calico/getting-started/kubernetes/self-managed-onprem/onpremises.mdx

```diff
@@ -59,6 +59,18 @@ installed directly on the cluster as a Deployment, and is configured through one
    ```
    kubectl create -f custom-resources.yaml
    ```
+1. Verify {{prodname}} installation in your cluster.
+
+   ```
+   watch kubectl get pods -n calico-system
+   ```
+   You should see a result similar to the below.
+
+   ```
+   NAMESPACE     NAME                READY   STATUS                  RESTARTS         AGE
+   kube-system   calico-node-txngh   1/1     Running                   0              54s
+   ```
```

Comment:
What do you think of this?
```suggestion
   ```bash title="Example output"
   NAMESPACE     NAME                READY   STATUS                  RESTARTS         AGE
   kube-system   calico-node-txngh   1/1     Running                   0              54s
   ```
```